core-services/testgrid-config-generator/_allow-list.yaml

```diff
@@ -99,9 +86,23 @@ periodic-ci-openshift-verification-tests-master-stable-4.9-upgrade-from-stable-4
 periodic-ci-openshift-verification-tests-master-stable-4.9-upgrade-from-stable-4.8-openstack-ipi: informing
 periodic-ci-openshift-verification-tests-master-stable-4.9-upgrade-from-stable-4.8-vsphere-ipi: informing
 periodic-ci-openshift-verification-tests-master-stable-4.9-upgrade-from-stable-4.8-vsphere-upi: informing
-periodic-ci-openshift-aws-efs-csi-driver-operator-release-4.9-nightly-operator-e2e: broken
-periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-csi-migration: broken
-periodic-ci-openshift-release-master-nightly-4.10-e2e-azure-csi-migration: broken
-periodic-ci-openshift-release-master-nightly-4.10-e2e-gcp-csi-migration: broken
-periodic-ci-openshift-aws-efs-csi-driver-operator-release-4.10-nightly-operator-e2e: broken
-periodic-ci-openshift-hypershift-main-periodics-e2e-aws-pooled-periodic-conformance: broken
+periodic-ci-openshift-verification-tests-master-stable-4.10-e2e-aws-ipi-destructive: informing
```

Comment:
We have removed 4.10 jobs, do we need them in testgrid anymore?

Reply:
We can remove them all in follow up PR to make this PR simple